local-cluster/bench/Main.hs

```diff
@@ -93,32 +105,37 @@ benchOptions =
           "Starts a cluster of Hydra nodes interconnected through a network and \
           \ talking to mock-chain, generates an initial UTxO set and a bunch \
           \ of valid transactions, and send those transactions to the cluster as \
-          \ fast as possible."
+          \ fast as possible.\n \
+          \ Arguments can control various parameters of the run, like number of ndoes, \
+          \ number of clients or type of transactions generater..."
         <> header "bench - load tester for Hydra node cluster"
     )
 
 main :: IO ()
 main =
   execParser benchOptions >>= \case
-    Options{outputDirectory = Just benchDir, scalingFactor, concurrency, timeoutSeconds, clusterSize} -> do
+    Options{outputDirectory = Just benchDir, scalingFactor, concurrency, timeoutSeconds, clusterSize, constantUtxo} -> do
       existsDir <- doesDirectoryExist benchDir
       if existsDir
         then replay timeoutSeconds clusterSize benchDir
-        else createDirectory benchDir >> play scalingFactor concurrency timeoutSeconds clusterSize benchDir
-    Options{scalingFactor, concurrency, timeoutSeconds, clusterSize} ->
-      createSystemTempDirectory "bench" >>= play scalingFactor concurrency timeoutSeconds clusterSize
+        else createDirectory benchDir >> play scalingFactor concurrency timeoutSeconds clusterSize constantUtxo benchDir
+    Options{scalingFactor, concurrency, timeoutSeconds, clusterSize, constantUtxo} ->
+      createSystemTempDirectory "bench" >>= play scalingFactor concurrency timeoutSeconds clusterSize constantUtxo
  where
+  play scalingFactor concurrency timeoutSeconds clusterSize constantUtxo benchDir = do
+    let generator = case constantUtxo of
```

Comment:
`constantUtxo` is not a good name IMO.. it is more a `scenario` or `transactionTypesToGenerate`